src/platforms/dotnet/common/unit-testing.mdx

```diff
@@ -1,22 +1,43 @@
 ---
-title: Unit Testing
-description: "Learn about unit testing with `ISentryClient` and `IHub`."
+title: Compatibility and Unit Testing
```

Comment:
Not sure it makes much sense to have a page for the two things.
They are quite unrelated.

Reply:
Ah, got it. Thanks.